src/scene/renderer/render-pass-cookie-renderer.js

```diff
@@ -1,8 +1,9 @@
+import { Debug } from '../../core/debug.js';
```

Comment:
What about a folder `renderer/render-passes`?

Reply:
maybe if we have more of those, yeah